app/hotels/src/HotelsReducer.js

```diff
@@ -1,8 +1,6 @@
 // @flow
 
-export type HotelsSearchParametersType = {|
-  latitude: number | null,
-  longitude: number | null,
```

Comment:
You were supposed to replace just these coordinates with city ID. Why is it so different now? Parameters to search hotels (`HotelsSearchParameters(Type)`) are: city ID, checkin, checkout, room configuration. This change makes the whole picture even more complicated. Can you please elaborate?

Reply:
Probably my fault by introducing QueryRenderer. I also removed completely local state away from SearchForm as it was redundant and now all settings are here.